content/en/developers/integrations/agent_integration.md

```diff
@@ -440,6 +440,33 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
 
 Once you have created your Agent-based integration, see the [Create a tile][20] documentation for information on populating the remaining [required assets][31] that appear on your integration tile, and opening a pull request.
 
+## Update your integration
+To update your integration, edit the relevant files and open a new pull request to your integration's directory in the [`integrations-extras`][21] or [`marketplace`][22] repository. 
+
+* If you are editing or adding new integration code, a version bump is required.
```

Comment:
Do you want to mention the diff between major and minor version bumps?

Reply:
discussed offline - going to resolve